bin/generate_client

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#!/bin/bash
```

Comment:
Why this change? I though the /usr/bin/env bash should help find the latest bash if it exists.

Reply:
I agree - would prefer to stick with `#!/usr/bin/env bash` generally

Reply:
Okay, I did it for consistencies sake with the rest of the scripts so I will change all of our bash scripts to use that version.